includes/Core/Authentication/Authentication.php

```diff
@@ -1464,9 +1464,11 @@ public static function invalid_nonce_error( $action ) {
 		$html  = __( 'The link you followed has expired.', 'google-site-kit' );
 		$html .= '</p><p>';
 		$html .= sprintf(
-			'<a href="%s">%s</a>',
+			'<a href="%s">%s</a>. <a href="%s" target="_blank">%s</a>.',
 			esc_url( Plugin::instance()->context()->admin_url( 'splash' ) ),
-			__( 'Please try again.', 'google-site-kit' )
+			__( 'Please try again', 'google-site-kit' ),
+			esc_url( $this->get_proxy_support_link_url() . '?error_id=nonce_expired' ),
+			__( 'Get Help', 'google-site-kit' )
```

Comment:
Can you change the `H` in help to a small caps `h` for consistency with the string used throughout the plugin?

Reply:
Updated.